package.json

```diff
@@ -11,14 +11,18 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "@prisma/client": "^5.11.0",
     "next": "12.0.10",
+    "npm": "^10.5.0",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-markdown": "8.0.0"
+    "react-markdown": "8.0.0",
+    "vercel": "^33.6.2"
```

Comment:
Wow this was the culprit? Good job finding it.

Reply:
yeah deleted the package-lock.json then npm installed again, after that it worked tbh I got lucky lol